src/background/adblocker.js

```diff
@@ -29,6 +29,7 @@ import asyncSetup from '/utils/setup.js';
 import { tabStats, updateTabStats } from './stats.js';
 import Config, {
   FLAG_FIREFOX_CONTENT_SCRIPT_SCRIPTLETS,
+  FLAG_CHROMEIUM_INJECT_COSMETICS_ON_RESPONSE_STARTED,
```

Comment:
"CHROMEIUM" -> "CHROMIUM"

However, I am not sure why this is only for Chrome, and if the flag is meant to be turned on only on Chrome, we still need to configure it; the name does not help, but might be confusing.

Reply:
The `chrome.webRequest.onResponseStarted` API is available in Firefox as well. Why do we only target Chromium?